README.md

```diff
@@ -109,6 +109,8 @@
 
 [x] Add the content for the first part of the workshop
 
+- Hynek is trying to propose this change.
```

Comment:
that is fine, but add a comment here as well

Reply:
That was cajk, bro.